### PR TITLE
fix(core): drop hasLabel property and handling the RTL issue in Status Indicator

### PR DESCRIPTION
**Pull request by Lokanathan-k**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/5901

#### Please provide a brief summary of this pull request.

addressing the breaking changes and handling the RTL issue

BREAKING CHANGE:
Removed [hasLabel] property as it was unnecessary.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/status-indicator/status-indicator.component.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <span
-    *ngIf="hasLabel && (labelPosition === 'left' || labelPosition === 'top')"
-    [ngClass]="'fd-status-indicator__label--' + labelSize"
+    *ngIf="labelPosition === 'left' || labelPosition === 'top'"
+    [ngClass]="{'fd-status-indicator__label--sm' : labelSize ==='sm',
+	'fd-status-indicator__label--md' : labelSize ==='md',
+	'fd-status-indicator__label--lg' : labelSize ==='lg',
+	'fd-status-indicator__label--xl' : labelSize ==='xl',
+	'fd-status-indicator__label--top': labelPosition === 'top',
+	'fd-status-indicator__label--left' : labelPosition === 'left'}"
```
Comment (InnaAtanasova) [inline libs/core/src/lib/status-indicator/status-indicator.component.html:8]: Not the approach we are using in fund-ngx. Please use:
```
'[class.className]': 'condition',
```

File: libs/core/src/lib/status-indicator/status-indicator.component.html [hunk 2]
```diff
@@ -16,8 +21,13 @@
...
 <span
-    *ngIf="hasLabel && (labelPosition === 'right' || labelPosition === 'bottom')"
-    [ngClass]="'fd-status-indicator__label--' + size"
+    *ngIf="labelPosition === 'right' || labelPosition === 'bottom'"
+    [ngClass]="{'fd-status-indicator__label--sm' : labelSize ==='sm',
+	'fd-status-indicator__label--md' : labelSize ==='md',
+	'fd-status-indicator__label--lg' : labelSize ==='lg',
+	'fd-status-indicator__label--xl' : labelSize ==='xl',
+	'fd-status-indicator__label--right': labelPosition === 'right',
+	'fd-status-indicator__label--bottom' : labelPosition === 'bottom'}"
```
Comment (InnaAtanasova) [inline libs/core/src/lib/status-indicator/status-indicator.component.html:30]: Same as above

File: libs/core/src/lib/status-indicator/status-indicator.component.ts [hunk 3]
```diff
@@ -75,12 +75,6 @@ export class StatusIndicatorComponent implements OnChanges, AfterViewInit, CssCl
     @Input()
     viewBox: string;
 
-    /**
-     * boolean value to define requirement of the label.
-     */
-    @Input()
-    hasLabel: boolean;
-
```
Comment (InnaAtanasova) [inline libs/core/src/lib/status-indicator/status-indicator.component.ts:83]: Why did you remove it?

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/status-indicator/status-indicator.component.ts (hunk 3 above)
Comment (Lokanathan-k) [reply libs/core/src/lib/status-indicator/status-indicator.component.ts:83]: not required

Comment (InnaAtanasova) [reply libs/core/src/lib/status-indicator/status-indicator.component.ts:83]: Should be documented in Breaking changes

Comment (Lokanathan-k) [reply libs/core/src/lib/status-indicator/status-indicator.component.ts:83]: Breaking changes has been updated

**Review by Betrozov — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

File: libs/core/src/lib/status-indicator/status-indicator.component.ts (hunk 3 above)
Comment (InnaAtanasova) [reply libs/core/src/lib/status-indicator/status-indicator.component.ts:83]: I don't see it in the PR template

Comment (InnaAtanasova) [reply libs/core/src/lib/status-indicator/status-indicator.component.ts:83]: Screen Shot 2021-07-22 at 11 01 25 AM

I added it

**Review by InnaAtanasova — APPROVED** [review]

**Comment by puru-hk** [comment]
Looks fine

**Review by puru-hk — APPROVED** [review]
